basic-examples/config-file/workflow.yaml

```diff
@@ -0,0 +1,18 @@
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties: {}
+
+steps:
+  ansible_facts:
+    plugin:
+      deployment_type: image
+      src: quay.io/arcalot/arcaflow-plugin-metadata:0.5.0
+    input: {}
+
+outputs:
+ success:
+   hostname: !expr $.steps.ansible_facts.outputs.success.env['HOSTNAME']
```

Comment:
It doesn't seem useful to me to mention "ansible" in this file:  in this context, we don't care how the plugin is implemented -- we care only that the step has an output accessible as `env['HOSTNAME']` which is returned upon successful completion.

I recommend naming the step to `metadata` or something similarly appropriate in context (and, obviously, fixing the reference in the output expression to match).